ios/RN/RNCamera.m

```diff
@@ -1284,7 +1295,7 @@ - (void)startSession
 
         AVCaptureStillImageOutput *stillImageOutput = [[AVCaptureStillImageOutput alloc] init];
         if ([self.session canAddOutput:stillImageOutput]) {
-            stillImageOutput.outputSettings = @{AVVideoCodecKey : AVVideoCodecJPEG};
+            stillImageOutput.outputSettings = @{AVVideoCodecKey : AVVideoCodecJPEG, AVVideoQualityKey: @(1.0)};
```

Comment:
What's this one exactly? Is this a change to video recording quality?

Reply:
Setting this key ensures the highest quality for the original taken image instead of relying on the iOS default value (which I could not find). The quality wanted for the resulting image is applied with these lines of code
```obj-c
float quality = [options[@"quality"] floatValue];
[metadata setObject:@(quality) forKey:(__bridge NSString *)kCGImageDestinationLossyCompressionQuality];
...
CGImageDestinationAddImage(destination, takenImage.CGImage, writeExif ? ((__bridge CFDictionaryRef) metadata) : nil);
```

By looking at this code, I think that I found a bug: The wanted quality is only applied if `writeExif` is not specified or is set to `true`. If a developer uses `writeExif: false` with `quality: 1.0` the resulting image has not the same quality/file size as if `writeExif: true` was used. Can anyone confirm this?

Reply:
You may be right about the exif bug.

About the above line change. That is interesting; by looking at `quality=1` photos size on an iphone 7 (3-4mb), I would say the quality was already 1. Is that something that can be tested easily? Now I wonder if the quality value should be applied to the above setting instead of the metadata.

Reply:
I've just tested, and using ` AVVideoQualityKey: @(1.0)` did not change anything to the file size on my test device (which is good, not a breaking change I guess, but it's better to be verbose about that setting).

About the exif bug, do you think you could fix it by just adding a "fake" metadata object with just the compression flag in the case exif is not used? Seems like a simple change as opposed to changing the value of `stillImageOutput.outputSettings` dynamically.

Reply:
Thank you for testing :) . I made a new commit https://github.com/react-native-community/react-native-camera/pull/2785/commits/55feaabe1f1fee5e689c6a9e83330771945b86f0 wich ensures that the compression flag is always specified.